src/main/kotlin/com/broeskamp/monorepo/gradle/plugin/kotlin/AndroidKotlinPlugin.kt

```diff
@@ -60,4 +68,25 @@ class AndroidKotlinPlugin : Plugin<Project> {
       }
     }
   }
+
+  protected fun Project.buildVersionCode(): Int = if (project.hasProperty("android.version.code")) {
+    project.property("android.version.code").toString().toInt()
+  } else {
+    val versionObject = Version.parse("$version")
+    var vc = versionObject.major * 100_000_000
```

Comment:
Ist das denn nun in der Reichweite?
Sagtest du nicht die Zahlen können dort nur bis 99.999.999 gehen?
Da wären wir ja schon drüber
Und was passiert, wenn die Version mal die 10.x.x überschreitet?

Reply:
So, wer nachschaut und es nicht aus dem 7 jähre alten memory holt (kopf!), der ist besser beraten!

Android sagt: The greatest value Google Play allows for versionCode is 2100000000.

Was nach die Version 2100.999.999 ergibt. Sollte also reichen.

Reply:
https://developer.android.com/studio/publish/versioning

Reply:
Okay also wenn finetrade irgendwann Version 20 erreicht, sollten wir uns langsam gedanken machen :D